### ISSUE TITLE
Feature Request: HDD Health Checker

### ISSUE THREAD

**Issue by AkoposiNobi** [aldostools/webMAN-MOD#1084]
a way to see HDD health within PS3 so that we can see if the drive is due for backup

thx a lot!

**Comment by aldostools** [comment]
You have to take the hdd out and check it with an application tool in the pc.

**Comment by Masamune3210** [comment]
Yeah, iirc, the required ioctl's arent exposed to the system to deal with SMART
